charts/pomerium/README.md

```diff
@@ -91,11 +91,32 @@ You may force recreation of your TLS certificates by setting `config.forceGenera
 
 If you wish to provide your own TLS certificates in secrets, you should:
 
-1. turn `generateTLS` to `false`
+1. turn `config.generateTLS` to `false`
 2. specify `authenticate.existingTLSSecret`, `authorize.existingTLSSecret`, and `proxy.existingTLSSecret`, pointing at the appropriate TLS certificate for each service.
 
 All services can share the secret if appropriate.
 
+## Signing Key
+
+### Auto Generation
+
+In default configuration, this chart will automatically generate a signing key in a helm `pre-install` hook for the Pomerium proxy to sign jwt sent in responses.
+
+Upon delete, you will need to manually delete the generated secret. Example:
+
+```console
+kubectl delete secret -l app.kubernetes.io/name=pomerium
```

Comment:
this is going to delete all secret with this label, is that the objective?

Reply:
Not really, we can have a more specific example like:

`kubectl delete secret -l name=pomerium-xxxxx-signing-key`

But the problem is the same with same example for deleting tls secrets. Should it be also more specific with example for each tls secret ?

Reply:
actually to delete a secret by name, use `kubectl delete secret pomerium-xxxxx-signing-key`.

> But the problem is the same with same example for deleting tls secrets. Should it be also more specific with example for each tls secret ?

I don't know, actually the example for the TLS secret is not correct anymore now that we have multiple secrets with this label…

Reply:
Basic secret (with `cookie-secret` and `shared-secret`) is also deleted by the example for TLS secret.

Should we add another annotation with secret type in order to show more selective examples ?

Reply:
We should probably update the docs to just reflect the names.  More labels sounds like overkill to me.

Reply:
done